### ISSUE TITLE
CocoaPods not installed error message from Android Studio but works fine in bash and terminal window in Android Studio...

### ISSUE THREAD

**Issue by dspells** [flutter/flutter-intellij#5727]
At this point I am assuming that the plugin is responsible for setting up the environment correctly in Android Studio so that it can find CocoaPods so that flutter doctor --verbose and flutter run will work properly.

Running / Debugging on an Android device seems to work perfectly. That makes me believe that both the Flutter and Dart plugins are installed and work properly when communicating to an Android device.

This is occurring in my Android Studio project. This may be just a setup problem but I have not been able to find an answer on the internet. If this is a simple setup issue I would appreciate being pointed in the right direction.

I know that CocoaPods is installed properly because it works perfectly from a bash shell command line. 'flutter run' on an iOS device works on the command line as well but it does not work in Android Studio.

The 'Flutter SDK' path is the same in both bash and Android Studio (see output of flutter doctor --verbose below).
I don't see any settings in 'Android Studio > Preferences > Languages & Frameworks > Flutter' that might make a difference except the Flutter SDK path.

This seems to very similar to if not the same as this closed bug https://github.com/flutter/flutter-intellij/issues/4981
I read through this bug and it didn't help. Somehow it just 'magically' started working for the person again.

iOS devices and emulators are recognized by the plugin so the 'flutter devices' must be working properly.

'which pod' returns the same answer in the command line as well as Android Studio terminal.

## Steps to Reproduce

In the bash command line type "flutter doctor --verbose"
If I open up "Tool Windows > Terminal" and run 'flutter doctor --verbose' it gets the same good answer.
flutter run to an iOS device also works properly inside the terminal window but not when I run from Android studio.
So what else is left but the plugin?
Output looks good and says
---
[✓] Flutter (Channel stable, 2.2.3, on macOS 11.5.2 20G95 darwin-x64, locale en-US)
    • Flutter version 2.2.3 at /Users/david.spells/flutter
    • Framework revision f4abaa0735 (8 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/david.spells/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /Users/david.spells/Library/Android/sdk
    • Java binary at: /Users/david.spells/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7621141/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.10.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Users/david.spells/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7621141/Android
      Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] IntelliJ IDEA Ultimate Edition (version 2021.2.1)
    • IntelliJ at /Users/david.spells/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin version 54.1.4
    • Dart plugin version 203.7759

[✓] VS Code (version 1.59.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.25.0

[✓] Connected device (3 available)
    • David D’s iPhone (mobile) • 321054e376491abda78e51f2324fa384c29837a4 • ios            • iOS 14.0.1
    • macOS (desktop)           • macos                                    • darwin-x64     • macOS 11.5.2 20G95 darwin-x64
    • Chrome (web)              • chrome                                   • web-javascript • Google Chrome 92.0.4515.159
---
In Android choose Tools > Flutter > Flutter Doctor
In the output it says that it can't find CocoaPods despite working properly on the command line.
---
/Users/david.spells/flutter/bin/flutter doctor --verbose
[✓] Flutter (Channel stable, 2.2.3, on macOS 11.5.2 20G95 darwin-x64, locale en-US)
    • Flutter version 2.2.3 at /Users/david.spells/flutter
    • Framework revision f4abaa0735 (8 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/david.spells/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Users/david.spells/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7621141/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[!] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Users/david.spells/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7621141/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] IntelliJ IDEA Ultimate Edition (version 2021.2.1)
    • IntelliJ at /Users/david.spells/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin version 54.1.4
    • Dart plugin version 203.7759

[✓] VS Code (version 1.59.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.25.0

[✓] Connected device (3 available)
    • David D’s iPhone (mobile) • 321054e376491abda78e51f2324fa384c29837a4 • ios            • iOS 14.0.1
    • macOS (desktop)           • macos                                    • darwin-x64     • macOS 11.5.2 20G95 darwin-x64
    • Chrome (web)              • chrome                                   • web-javascript • Google Chrome 92.0.4515.159

! Doctor found issues in 1 category.
---

## Version info

The output of flutter doctor --verbose for both the command line and Android Studio is in the 'steps to reproduce section'.

**Comment by stevemessick** [comment]
tl;dr: `chsh zsh`

See `man chsh`. Unless you tell MacOS that you want to use a different shell it will still use `bash`. For new accounts, the default is `zsh` but transferred accounts will use the previous setting.
